bin/test-addon-pr.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # assumptions
 # - will only run the latest version of an addon if there are multiple version changes (latest is simple dictionary sort :/ )
-# - will run all addon changes in a single spec if multiple addons were changed.
+# - will each addon change as a separate testgrid run. Testing two addon updates simultaneously is not supported.
```

Comment:
`will each addon change as a separate testgrid run`

Reply:
maybe `will run each addon change independently`?

Reply:
Going with `will run each changed addon as a separate testgrid run. Testing two addon updates simultaneously in a TestGrid run is not supported.
`